.template/hooks/before_complete/report.rb

```diff
@@ -0,0 +1,27 @@
+def report
+  after_bundle do
+    say "\n🚀 The project has successfully created 🚀\n", :green
+
+    # Report errors
+    unless @template_errors.empty?
+      print_separator
+      say "\n📍 There are some errors when templating the application, Please fix them manually:\n"
```

Comment:
Minor issue with capitalization and tense:

```suggestion
      say "\n📍 There were some errors when templating the application, please fix them manually:\n"
```

Reply:
🙈  Fixed all typo on [1907339](https://github.com/nimblehq/rails-templates/pull/317/commits/19073399774772efc3ef58fd546751f55bb846cb) ✨